runtime/src/main/scala/akka/grpc/javadsl/ServiceHandler.scala

```diff
@@ -17,9 +18,17 @@ import scala.annotation.varargs
 @ApiMayChange
 object ServiceHandler {
 
+  /**
+   * INTERNAL API
+   */
+  @InternalApi
   private[javadsl] val notFound: CompletionStage[HttpResponse] =
     CompletableFuture.completedFuture(HttpResponse.create().withStatus(StatusCodes.NOT_FOUND))
 
+  /**
+   * INTERNAL API
+   */
+  @InternalApi
   private[javadsl] val unsupportedMediaType: CompletionStage[HttpResponse] =
     CompletableFuture.completedFuture(HttpResponse.create().withStatus(StatusCodes.UNSUPPORTED_MEDIA_TYPE))
```

Comment:
`notFound` and `unsupportedMediaType` are `CompletionStage` specific here because this is `javadsl`. In the `scaladsl` version of `ServiceHandler` there is also a `notFound` and a `unsupportedMediaType` but those are `Future[HttpRespomse]`, ofc. Note I'm only annotating `InternalApi` the `javadsl` copies on purpose.

(off-topic) Each `MyServiceHandlerFactory.{java.scala}` class generated using the [Java](https://github.com/akka/akka-grpc/blob/817aa17258604838ae93f809f52a5aff8fd251a6/codegen/src/main/twirl/templates/JavaServer/Handler.scala.txt#L36-L40)/[Scala](https://github.com/akka/akka-grpc/blob/817aa17258604838ae93f809f52a5aff8fd251a6/codegen/src/main/twirl/templates/ScalaServer/Handler.scala.txt#L29-L30) `Handler.scala.txt` templates, also has its own pair of `notFound` and `unsupportedMediaType` constants.

This sounds like Akka-gRPC (or even Akka-HTTP) could use a collection of already completed `CS/Future` responses for `notFound` and `unsupportedMediaType` which we could then use from generated code of `ServiceHandler`.

NOTE: the copies in `ServiceHandler` were added in #829.

Reply:
raised #930